impeller/golden_tests/golden_playground_test_mac.cc

```diff
@@ -128,8 +128,7 @@ void GoldenPlaygroundTest::SetUp() {
   std::filesystem::path icd_path = target_path / "vk_swiftshader_icd.json";
   setenv("VK_ICD_FILENAMES", icd_path.c_str(), 1);
 
-  if (GetBackend() != PlaygroundBackend::kMetal &&
-      GetBackend() != PlaygroundBackend::kVulkan) {
```

Comment:
This should be turning off vulkan golden images, which isn't what we want.

Reply:
Fixed?